### PR TITLE
Add pagination to search

### PR DESCRIPTION
**Pull request by murilovmachado**
Jira: https://bugs.benetech.org/browse/CHI-227

This PR:
- Adds Sequelize Pagination to `contacts/search`
- Extract `getPaginationElements` helper
- Updates Swagger
- Update tests

### CODE REVIEW

**Review by nickhurlburt — COMMENTED** [review]
Looking good, and thanks for updating swagger as well!

File: controllers/case-controller.js [hunk 1]
```diff
@@ -26,12 +26,8 @@ const CaseController = Case => {
   };
 
   const listCases = async query => {
-    const queryLimit =
-      query.limit && !Number.isNaN(parseInt(query.limit, 10))
-        ? parseInt(query.limit, 10)
-        : Infinity;
-    const limit = Math.min(queryLimit, 1000);
-    const offset = (query.offset && parseInt(query.offset, 10)) || 0;
+    const { limit, offset } = getPaginationElements(query);
```
Comment (GPaoloni) [inline controllers/case-controller.js:29]: Clean :sunglasses:

**Review by GPaoloni — APPROVED** [review]
Looks and works great! :rocket:

**Comment by murilovmachado** [comment]
Merged, but I'll only deploy it when the front end is done.

**Pull request merged**